packages/angular/src/editor/editor.component.ts

```diff
@@ -370,6 +370,7 @@ export class EditorComponent extends RootComponent implements OnInit, TriggerInt
     froalaConfig: LooseObject = {
         attribution: false,
         key: Stratus.Environment.get('froalaKey'),
+        zIndex: 9999,
```

Comment:
I'm assuming you don't expect any popups to appear over the editor

Reply:
Presumably, the `zIndex` field is only used for their tooltips.  I have an Angular+ Dialog for the Media Selection that goes over the entire editor without any problems.